src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BAAL_COUNT = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(BAAL_COUNT);
```

Comment:
`BAAL_COUNT` is the typo here?

Reply:
Yep!) BAAL is a demon and I wanted to write "BALL". Corrected the variable name.